bin/disco_aws.py

```diff
@@ -266,29 +296,35 @@ def run():
             ami_dict = bake.list_amis_by_instance(instances)
             now = datetime.utcnow()
 
+        header = format_listhosts_header(args, most)
+        divider = '-'.ljust(len(header), '-')
+
+        print("\n" + header + "\n" + divider)
```

Comment:
Recommendation: check if stdout is a TTY, print only in that case.  Screws things up if people are trying to print to a file, but avoids weird things appearing on STDERR when you have a pipeline.